DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -337,7 +337,12 @@ private function odmLoad(array $config, ContainerBuilder $container)
 
             $container->setParameter('doctrine_phpcr.odm.locales', $config['locales']);
             $container->setParameter('doctrine_phpcr.odm.allowed_locales', array_keys($config['locales']));
-            $container->setParameter('doctrine_phpcr.odm.default_locale', key($config['locales']));
+            if (isset($config['default_locale']) && ! is_null($config['default_locale'])) {
+                $defaultLocale = $config['default_locale'];
```

Comment:
we could sanity check that the default_locale indeed exists in the keys of $config['locales'] and throw an invalid configuration exception otherwise.